web/react/utils/utils.jsx

```diff
@@ -828,11 +828,11 @@ export function isValidUsername(name) {
     } else if (name.length < 3 || name.length > 15) {
         error = 'Must be between 3 and 15 characters';
     } else if (!(/^[a-z0-9\.\-\_]+$/).test(name)) {
-        error = "Must contain only lowercase letters, numbers, and the symbols '.', '-', and '_'.";
+        error = "Must contain only letters, numbers, and the symbols '.', '-', and '_'.";
     } else if (!(/[a-z]/).test(name.charAt(0))) {
         error = 'First character must be a letter.';
     } else {
-        var lowerName = name.toLowerCase().trim();
+        var lowerName = name;
```

Comment:
Is this necessary? If `lowerName` is the same as `name` can we just use `name` and not create a new variable?

I know it's more code change but I'd prefer to avoid unnecessary variables :)